DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -1490,30 +1490,32 @@ public async Task<DiscordMember> GetMemberAsync(ulong userId, bool updateCache =
         CancellationToken cancellationToken = default
     )
     {
-        int recievedLastCall = 1000;
         ulong last = 0ul;
-        while (recievedLastCall > 0)
+        for (int i = MemberCount; i > 0;)
```

Comment:
Im not a friend of using the member count, discord uses eventuall consitency for many counters. I think only to stop if the last call contains less than the limit (1000) is the best way to stop the loop. Also this would be a 1 in a 1000 case

Reply:
Also a cache thing. Our stored member count could be stale data

Reply:
I do agree with this, I suggested that as well.
This solution was brought up by @VelvetToroyashi with some input from @akiraveliara [in discord](https://discord.com/channels/379378609942560770/379386901725052928/1230279194345209937). I can change it to that if that's what you guys agree on.

Reply:
I'm also against this change. While the concept is a great idea, Discord said no.

Reply:
I'm also against this change. While the concept is a great idea, Discord said no.